src/postgres/postgresConnectionStrings.ts

```diff
@@ -35,13 +40,19 @@ export class ParsedPostgresConnectionString extends ParsedConnectionString {
     public username: string;
     public password: string;
     public readonly port: string;
+    public readonly sslSupport: boolean;
 
     constructor(connectionString: string, config: ConnectionOptions) {
         super(connectionString, config.database ? config.database : undefined);
         this.hostName = nonNullProp(config, 'host');
         this.port = config.port ? config.port : `${postgresDefaultPort}`;
         this.username = nonNullProp(config, 'user');
         this.password = nonNullProp(config, 'password');
+        if (config.ssl === false) {
+            this.sslSupport = false;
+        } else {
+            this.sslSupport = true;
```

Comment:
This is assuming ssl is supported in all other cases --> config.ssl = string | undefined | true.